config.json

```diff
@@ -252,7 +252,13 @@
       "core": false,
       "difficulty": 4,
       "slug": "kindergarten-garden",
-      "topics": null,
+      "topics": [
+          "loops",
+          "arrays",
+          "strings",
+          "logic",
+          "pattern_recognition"
```

Comment:
Great suggestions! :) I would add `enumerations` and `lists` as well, since they're used in the tests. Do you agree? :)

Also, these should be indented two spaces in from `topics` :)

Reply:
Yeah, great idea. Add them too.

Reply:
Is there something else I have to do?

Reply:
You should add `enumerations` and `lists` to the topics :)

Reply:
And you need to fix the indentation like I described. You can make those changes in a new commit and push them so that it gets added to this PR. Does that make sense?

Reply:
On my repo?

Reply:
Yes, on your repo. If you push to the same branch then it will automatically get added to this PR, so you shouldn't open another PR. This is why it's easier to do PR work on a branch because then you can keep working on other things and only push things on that branch if you want them to be added to the PR. In your case you can make the new commit and push it on your `master` branch and it will get added to this PR :) Does that make sense?

Reply:
Okay.